docs/.vuepress/components/Button.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <!-- <h1>Test</h1> -->
+  <router-link v-if="isRouter" :to="to" :class="buttonClasses">
+    <slot />
+  </router-link>
+  <a href v-else-if="isA" :class="buttonClasses" :target="target" :rel="rel">
+    <slot />
+  </a>
+</template>
+
+<script>
+export default {
+  name: 'Button',
+  props: {
+    isA: {
+      type: Boolean,
+      default: true
+    },
+    isRouter: {
+      type: Boolean,
+      default: false
+    },
+    to: {
+      type: String
+    },
+    newTab: {
+      type: Boolean,
+      default: false
+    },
+    primary: {
+      type: Boolean,
+      default: true
+    },
+    secondary: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    buttonClasses() {
+      if (this.secondary) {
+        return 'button secondary'
+      } else {
+        return 'button primary'
+      }
+    },
+    target() {
```

Comment:
These `target` & `rel` computed properties could just be put directly in the component, i.e.
```
  <a href v-if="isExternal" :class="buttonClasses" :target="_blank" rel="noopener noreferrer">
    <slot />
  </a>
  <router-link v-else :to="to" :class="buttonClasses">
    <slot />
  </router-link>
```